assets/scss/_styles_project.scss

```diff
@@ -110,28 +110,31 @@
     font-size: 18px;
     margin-bottom: 0;
   }
+
   img {
     margin-top: 1rem;
     width: 20rem;
     max-width: 80%;
   }
+
+  a.external-link:after {
+    display: none;
+  }
 }
 
-.td-box--dark p > a {
-  color: #92b5e7;
+.td-box--dark a {
+  color: lighten($primary, 25%) !important;
```

Comment:
Why is there a need for using `!important` ?

Reply:
To override the Docsy default which is applied via a more specific rule that uses `p >`. We want the override to apply to all links inside a dark box.